docs/_includes/components/input-groups.html

```diff
@@ -26,6 +26,7 @@ <h2 id="input-groups-basic">Basic example</h2>
   <p>Place one add-on or button on either side of an input. You may also place one on both sides of an input.</p>
   <p><strong class="text-danger">We do not support multiple add-ons on a single side.</strong></p>
   <p><strong class="text-danger">We do not support multiple form-controls in a single input group.</strong></p>
+  <p><strong class="text-danger"><code>input-group-addon</code> must be the last child within an input group.</strong></p>
```

Comment:
That's a fantastic question. No clue where this even came from as I haven't opened that file